CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
   - Update to reqwest 0.11.0
   - Update to bytes 1.0
   - Require minimum protobuf of 2.20.0
+- [[#215](https://github.com/IronCoreLabs/ironoxide/pull/215)]
+  - Update to recrypt 0.12.0
+  - Bump MSRV to 1.41.1
```

Comment:
Isn't this only pertinent to our build process? I think it might be best to omit that from the changelog.

Reply:
MSRV applies to anyone depending on ironoxide. In other words, if you want to depend on ironoxide, you must be using rustc 1.41.1 or higher.

Reply:
Are you referring to the MSRV? That's the minimum version of rust that we say we support (and test against in CI), so I think it's important to include that for others to know.

Reply:
OK - agreed.